source/views/guides/getting-started.html.slim

```diff
@@ -206,6 +206,19 @@ main
               If you want to use SQLite instead, you need to use `INTEGER`
               instead.
 
+        aside.aside.aside--note
+          header.aside__header A Note on Using Migrations in Production
+          .aside__text
+            markdown:
+              When preparing your app for use in production, you may want to run your migrations during
+              the application's initialization phase. You may also want to include the migration scripts
+              as a part of your code, to avoid having to copy them to your deployment location/image etc.
+
+              The `diesel_migrations` crate provides the `embed_migrations!` macro, allowing you to embed
```

Comment:
This should link to the actual documentation of this crate/macro

Reply:
@weiznich done!